pkg/kine/drivers/generic/generic.go

```diff
@@ -20,7 +20,10 @@ import (
 	"go.opentelemetry.io/otel/trace"
 )
 
-const otelName = "generic"
+const (
+	otelName            = "generic"
+	defaultMaxIdleConns = 2 // default from database/sql
```

Comment:
is there a reason you use 5 in the CLI?

Reply:
Five is our current default value.

Reply:
It probably helps to bump this number for large clusters. But that should be possible on bootstrap through this change now.